app/src/main/java/org/mozilla/reference/browser/concepts/HistoryStorage.kt

```diff
@@ -26,4 +27,12 @@ interface HistoryStorage : HistoryStorage {
      * Remove domains from the 'block list' which are blocked from top sites.
      */
     fun restoreTopSites()
+
+    suspend fun getBookmarks(): JSONArray
```

Comment:
Why a JSONArray?

Reply:
That's what our DB returned before. I am converting it to an object when I am using it. 

See BookmarkUseCases.kt

https://github.com/cliqz/daisy/blob/692649626e42608fe9aa01a5474350df6aea3702/app/src/main/java/org/mozilla/reference/browser/library/bookmarks/usecases/BookmarkUseCases.kt#L20